Initialize as an R project

johpiip opened the pull request:
In this PR, this repository will be initialized as an R project. Writing scripts with other programming languages, e.g. Python, is still fine - this PR only helps R to run smoothly.

`renv` is a package that controls package dependencies in R in a similar manner to `pipenv` in Python. Moreover, the MRAN mirror takes care that all packages and their dependencies come from a same point in time to avoid dependency collisions.

zptro reviewed (approved):
Great that we handle dependencies and imports in a structured way.

johpiip commented:
Added https://github.com/HSLdevcom/mal-effect-calculations/pull/1/commits/807bec88ad6ff4032c6061b71e8e2c5a1dfea084 where I updated instructions on running the scripts with `renv`. No more commits to this branch!
